src/test/java/sqlline/SqlLineArgsTest.java

```diff
@@ -2225,6 +2228,91 @@ public void testStartupArgsWithoutUrl() {
     }
   }
 
+  /**
+   * Test to Set/Reset confirm.
+   */
+  @Test
+  public void testConfirmFlag() {
+    DispatchCallback callback = new DispatchCallback();
+    SqlLine line = new SqlLine();
```

Comment:
Could you please use custom output stream here, e.g. like at `sqlline.SqlLineArgsTest#testSetWrongColorScheme`
it will keep output clearer in case of success tests

Reply:
I believe it is related to last executed command? This test is intended to make sure variable value is being set appropriately.  Am I missing something?

Reply:
Yes, sure, you can look at `sqlline.SqlLineArgsTest#testSetWrongColorScheme` as the example where a custom output stream is used.